h2o-docs/src/product/data-science/algo-params/single_node_mode.rst

```diff
@@ -0,0 +1,42 @@
+``single_node_mode``
+--------------------
+
+- Available in: CoxPH, Deep Learning
+- Hyperparameter: yes
```

Comment:
This is not a hyperparameter.
An option is a hyperparameter only if the schema shows it as "gridable=true", which is not the case with this option:
https://github.com/h2oai/h2o-3/pull/4451/files#diff-bf97f6406bbd14646fa7194138b45efe